.github/workflows/dotnet.yml

```diff
@@ -0,0 +1,139 @@
+      shell: powershell
+      
+    - name: Integration Tests
+      run: |
+        echo ${{ github.workspace }}
+        
+        Write-Host "Creating the temporary CDR directory which holds the logs and database file..."
+        New-Item -ItemType Directory -Path d:\a\mock-register\mock-register\cdr -Force
+        
+        Write-Host "Running the solutions..."
+        cd ${{ github.workspace }}\Source\CDR.Register.API.Gateway.TLS
+        Start-Process dotnet -ArgumentList "run -v m -c ${{ env.buildConfiguration }} --launch-profile CDR.Register.API.Gateway.TLS.Pipeline"
+        
+        cd ${{ github.workspace }}\Source\CDR.Register.API.Gateway.mTLS
+        Start-Process dotnet -ArgumentList "run -v m -c ${{ env.buildConfiguration }} --launch-profile CDR.Register.API.Gateway.mTLS.Pipeline"
+        
+        cd ${{ github.workspace }}\Source\CDR.Register.Admin.API
+        Start-Process dotnet -ArgumentList "run -v m -c ${{ env.buildConfiguration }} --launch-profile CDR.Register.Admin.API.Pipeline"
+        
+        cd ${{ github.workspace }}\Source\CDR.Register.Discovery.API
+        Start-Process dotnet -ArgumentList "run -v m -c ${{ env.buildConfiguration }} --launch-profile CDR.Register.Discovery.API.Pipeline"
+        
+        cd ${{ github.workspace }}\Source\CDR.Register.SSA.API
+        Start-Process dotnet -ArgumentList "run -v m -c ${{ env.buildConfiguration }} --launch-profile CDR.Register.SSA.API.Pipeline"
+        
+        cd ${{ github.workspace }}\Source\CDR.Register.Status.API
+        Start-Process dotnet -ArgumentList "run -v m -c ${{ env.buildConfiguration }} --launch-profile CDR.Register.Status.API.Pipeline"
+        
+        cd ${{ github.workspace }}\Source\CDR.Register.IdentityServer
+        Start-Process dotnet -ArgumentList "run -v m -c ${{ env.buildConfiguration }} --launch-profile CDR.Register.IdentityServer.Pipeline"
+        
+        cd ${{ github.workspace }}
+        
+        Write-Host "Sleeping for 30 seconds to allow solutions to startup..."
+        Start-Sleep -s 30
+        
+        Write-Host "Listing contents of d:\a\mock-register\mock-register\cdr directory..."
+        cd d:\a\mock-register\mock-register\cdr
+        dir
+        
+        Write-Host "Basic connectivity testing..."
+        [System.Net.ServicePointManager]::ServerCertificateValidationCallback = { $True }
+        
+        try
+        {
+          Write-Host "Testing https://localhost:7000/idp/.well-known/openid-configuration..."
+          $apiReq = [System.Net.HttpWebRequest]::CreateHttp("https://localhost:7000/idp/.well-known/openid-configuration")
+          $apiRes = $apiReq.GetResponse()
+          Write-Host $apiRes.StatusCode
+        }
+        catch
+        {
+          Write-Host "Web request failed"
+          $log = Get-Content "d:\\a\\mock-register\\mock-register\\cdr\\Logs\\cdr-mr-tls-gateway.log" -Raw
```

Comment:
update to use the ${{ github.workspace }} var instead of direct linking the path

Reply:
This has been done.  All hardcoded references to d:\a\mock-register\mock-register in the yml file.